langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -141,6 +205,10 @@ public isolated function includes(string str, string substr, int startIndex = 0)
 
 # Tests whether a string starts with another string.
 #
+# ```ballerina
+# "Welcome to Ballerina programming language".startsWith("Welcome") ⇒ true
```

Comment:
Shouldn't this be

```suggestion
# "Welcome to the Ballerina programming language".startsWith("Welcome") ⇒ true
```

Or can't we just use something like

```suggestion
# "The Ballerina programming language".startsWith("The") ⇒ true
```

Reply:
I prefer the first option than the second :)